src/midiEditorCommands.cpp

```diff
@@ -539,6 +634,15 @@ vector<MidiNote> getSelectedNotes(MediaItem_Take* take, int offset=-1) {
 	return notes;
 }
 
+using MidiControlChangeIterator = MidiEventIterator<MidiControlChange>;
+
+template<>
+int MidiControlChangeIterator::getCount() const {
```

Comment:
I assume you're planning on using this pretty soon? if not, I'd suggest removing from this PR and adding this in future.

Reply:
It is needed for #599, so yes.